app/jobs/score_updater_job.rb

```diff
@@ -1,22 +1,22 @@
 class ScoreUpdaterJob
   include SuckerPunch::Job
 
-  def perform(user_id, score=1)
-    add_score(user_id, score, DailyScore)
-    add_score(user_id, score, WeeklyScore)
-    add_score(user_id, score, MonthlyScore)
-    add_score(user_id, score, OverallScore)
+  def perform(user_id, delta=1)
+    add_delta(user_id, delta, DailyScore)
+    add_delta(user_id, delta, WeeklyScore)
+    add_delta(user_id, delta, MonthlyScore)
+    add_delta(user_id, delta, OverallScore)
   end
 
-  def add_score(user_id, score, score_klass, expires_at=nil)
+  def add_delta(user_id, delta, score_klass, expires_at=nil)
```

Comment:
I liked as it was so `add_score(user_id, score, ...)`. I think it reads better than delta.